### PR TITLE
Add 2020 election process doc

### PR DESCRIPTION
**Pull request by bhs**
This is meant to be a straight port of the following google doc:

  https://docs.google.com/document/d/1SyTo67KG3HuvwS1a7jDdbXD52Raru01M3QSBjpJ7OMI/edit#

I've also moved over the structure of the 2019 candidate list since we are using PRs for nominations this time around.

See also: #488 

cc: @alolita @kateetak @SergeyKanzhelev

### CODE REVIEW

**Comment by kateetak** [comment]
Lgtm. Happy to re-review once the todos are filled.

**Comment by kateetak** [comment]
One more idea: as part of the "Results are announced" task (or even as a separate one), it would be good to also have the list of the new members published here: https://github.com/open-telemetry/community/blob/master/governance-charter.md

When I started browsing this repo, it took me some time to realize that the elections happened last year until I found the blog post on Medium: https://medium.com/opentelemetry/welcome-new-opentelemetry-governance-committee-members-6ac09fee3777

File: elections/2020/governance-committee-election.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+One of two options will be available in the form to prove eligibility:
+
+* GitHub handle
+* List either…
+  * Your contributions to OpenTelemetry that make you eligible to vote, or
+  * Your non-code contributions per the [Members of Standing section of the
+charter](https://github.com/open-telemetry/community/blob/master/governance-charter.md#members-of-standing):
+particularly, your relationship to a well-known project that’s taken a hard
+dependency on OpenTelemetry, and/or your involvement with a well-known
+organization’s adoption of OpenTelemetry as an end-user.
```
Comment (SergeyKanzhelev) [inline elections/2020/governance-committee-election.md:115]: styling nit: I prefer spacing in lists paragraphs. It renders OK on GitHub, but may not render well on other platforms =)

```suggestion
    charter](https://github.com/open-telemetry/community/blob/master/governance-charter.md#members-of-standing):
    particularly, your relationship to a well-known project that’s taken a hard
    dependency on OpenTelemetry, and/or your involvement with a well-known
    organization’s adoption of OpenTelemetry as an end-user.
```

File: elections/2020/governance-committee-election.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+* Nominees ratified and Governance Committee ensures nominees’ commitment
+* October 19: List of nominees is finalized on [community](https://github.com/open-telemetry/community) GitHub and advertised via mailing list and Gitter
+* October 25: End of call for voters pre-registration (registration during the voting period is permitted, but highly discouraged)
+* October 26, 12:01am GST: Emails with voting links sent via [CIVS](https://civs.cs.cornell.edu/). Voting period begins
+* October 28, 12:01am GST: Voting ends
+* October 30: Results are announced
+
+# Appendix
+
+We identified active contributors. For these contributors we found at least one
```
Comment (SergeyKanzhelev) [inline elections/2020/governance-committee-election.md:168]: we need to as Lucasz when he can generate the list for us and declare this date. So we count eligibility based on number of contributions by specific date and exactly one year back.

**Review by SergeyKanzhelev — APPROVED** [review]
I think it's OK if you want to merge this doc as is and fill out TODOs - then it would't be blocked on you=)

File: elections/2020/governance-committee-election.md (hunk 2 above)
Comment (bhs) [reply elections/2020/governance-committee-election.md:168]: ok cool. I guess we don't formally need that for the first phase of this (that starts tomorrow) which is about getting candidates together.

**Comment by bhs** [comment]
I'm going to merge this... happy to accept PRs if there are more things that need changing. (I think it'll be easier to discuss diffs that way)

**Pull request merged**